config/deploy.rb

```diff
@@ -13,4 +13,14 @@
 end
 
 before "deploy:setup", 'paperclip:create_storage'
-after "deploy:update_code", "paperclip:link_storage"
\ No newline at end of file
+after "deploy:update_code", "paperclip:link_storage"
+
+# Symlink the database
+after 'deploy:update_code', 'deploy:symlink_db'
+
+namespace :deploy do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.